### PR TITLE
Adding name and checkbox to newsletter

### PR DESCRIPTION
**Pull request by Gabriel637**
What problem is this solving?
New inputs on newsletter

How to test it?
Add showName, placeholderName on newsletter declaration

https://newsletter--motorolaes.myvtex.com/

Screenshots or example usage:

image

*need to link store-graphql pull request "adding firstName to newsletter"*

### CODE REVIEW

File: docs/Newsletter.md [hunk 1]
```diff
@@ -42,6 +42,10 @@ Through the Storefront, you can change the `Newsletter`'s behavior and interface
 | `placeholder` | `String` | Placeholder of the email input | `Enter your email address` (translated text) |
 | `submit` | `String` | Label of the submit button | `Sign up` (translated text) |
 | `hideLabel` | `Boolean` | Hide label | `false` |
+| `showName` | `Boolean` | Show name input | `false` |
+| `placeholderName` | `String` | Placeholder of the name input | `""` |
+| `showTerms` | `Boolean` | Show checkbox for terms | `false` |
+| `checkboxText` | `String` | Placeholder of the terms checkbox | `""` |
```
Comment (guerreirobeatriz) [inline docs/Newsletter.md:48]:
```suggestion
| `showName` | `boolean` | Whether the name input should be displayed (`true`) or not (`false`). | `false` |
| `placeholderName` | `string` | Name input text string | `null` |
| `showTerms` | `boolean` | Whether a checkbox for terms should be displayed (`true`) or not (`false`).  |  `false` |
| `checkboxText` | `string` | Text string for terms checkbox. | `null` |
```

Comment (guerreirobeatriz) [reply docs/Newsletter.md:48]: Don't know if I correctly understood the idea for `name input`. What is it?

Comment (Gabriel637) [reply docs/Newsletter.md]: the input for the name of the user, some stores are already using some kind of custom component to get the name and the email

**Comment by klzns** [comment]
We created the app https://github.com/vtex-apps/store-form, that solves this issue

**Pull request closed without merging**